addOns/replacer/src/main/javahelp/org/zaproxy/zap/extension/replacer/resources/help/contents/replacer.html

```diff
@@ -54,9 +54,12 @@ <h3>Match String</h3>
 <h3>Match Regex</h3>
 If set then the Match String will be treated as a regex expression.
 This option is disabled when matching actual headers.
+Hexadecimal bytes are represented with "\x00 - \xFF" (for example "abc\x01\x02\x03def").
 
 <h3>Replacement String</h3>
 The new string that will replace the specified selection.
+Hexadecimal bytes are represented with "\x00 - \xFF".
+"\x00" byte can be keept with "\\x00".
```

Comment:
`The literal string "\x00" can be kept...`?

Reply:
I still think this needs to be tweaked, that's not a byte but a string.

Reply:
Oh, ok! I thought it was all about *kept* instead of *keept*
I didn't notice the "string".